dusty/commands/test.py

```diff
@@ -103,7 +103,7 @@ def _services_compose_up(expanded_specs, app_or_lib_name, testing_spec, suite_na
         previous_container_names.append("{}_{}_1".format(_compose_project_name(app_or_lib_name, suite_name), service_name))
     return previous_container_names
 
-def _app_or_lib_compose_up(testing_spec, app_or_lib_name, app_or_lib_volumes, test_command, previous_container_name, suite_name):
+def _app_or_lib_compose_up(base_compose_spec, app_or_lib_name, app_or_lib_volumes, test_command, previous_container_name, suite_name):
```

Comment:
testing_base_compose_spec?  I thought this was at the level of gcweb, not gcweb.tests.  This could just be me though